ocs_ci/ocs/device_classes.py

```
@@ -30,6 +41,180 @@
+        sleep,
+        _is_deviceset_count_healed,
+        deviceset_name,
+        target_count,
+    ):
+        if healed:
+            log.info(
+                "Deviceset %s heal completed: count=%s, no non-Bound PVCs",
+                deviceset_name,
+                target_count,
+            )
+            return True
```

Ruff (0.16.1)

[warning] Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)